scene/gui/progress_bar.h

```diff
@@ -63,15 +66,20 @@ class ProgressBar : public Range {
 	};
 
 	void set_fill_mode(int p_fill);
-	int get_fill_mode();
+	int get_fill_mode() const;
```

Comment:
You need to add a compatibility method for this if you change it here, I'd leave it as is